Source/Classes/PINURLSessionManager.h

```diff
@@ -13,6 +13,7 @@ extern NSString * __nonnull const PINURLErrorDomain;
 @protocol PINURLSessionManagerDelegate <NSObject>
 
 @required
+- (void)didReceiveResponse:(nonnull NSURLResponse *)response forTask:(nonnull NSURLSessionTask *)task;
```

Comment:
Because of [this](https://github.com/pinterest/PINRemoteImage/pull/331/files#diff-ad3073891c160428009d09e3f7d1efbdR176), does it make sense for this method to be optional?